README.md

```diff
@@ -5,18 +5,30 @@ Make sure that you're accepted the invitation to become contributor of this repo
 To contribute:
 
 1. Clone the repository
-    ```bash
-    git clone git@github.com:project-zusammen/bookswap-frontend.git
-    ```
+
+   ```bash
+   git clone git@github.com:project-zusammen/bookswap-frontend.git
+   ```
 
 2. Checkout to a feature branch, add and commit your changes
-    ```bash
```

Comment:
kalau masih ada changes kaya gini berarti belum sama ya sama kondisi awalnya.

Ini kayanya kamu lupa kasih tab di awal line.

kalau sama kaya awalnya dia akan terlihat kaya line 1 - 7, no changes.

Buat skrang gapapa, aku approve. Next time kalau ada unnecessary changes tolong di revert. Thank you

Reply:
Feel free to merge the PR